content/articles/switch-vs-router-vs-hub/index.md

```diff
@@ -0,0 +1,117 @@
+In networking the terms "Switches", "Hubs", and "Routers" are sometimes used interchangeably and this is wrong. Despite them being similar, there are differences in how they handle data. These three components may be integrated into a single device making it hard for a student to distinguish between them.
+
+In this article, we are going to discuss each device and their functionalities in a network.
+
+### Switch
+A switch is a multicast networking device that works under the Datalink layer of the OSI model and connects a bunch of computers or devices in a network.
+

```

Comment:
The definition of a switch is too summarized. Please include some more information